### DISCUSSION TITLE
Contributor's background, expertise, context, ambition, availability

### DISCUSSION THREAD

**Discussion by djeanner** [TheELNConsortium/TheELNFileFormat#48] [Q&A]
Hi, I think that if each of us could answer the following questions, it would be very usefull to evaluate how to contribut to this project.

- My background and expertise relevant to the goal of the ELNConsortium ...
- My current activities and their relevance to the ELNConsortium ...
- My contribution to the ELNconsortium goal could be ...
- My short-term activity within the ELNConsortium include ...

**Comment by NicolasCARPi** [comment]
I'm the founder of the consortium with @SteffenBrinckmann. We try to impulse movement to the consortium (like organizing meetings). Ideally, decisions are made collegially so we aim to have a role of moderator/animator in the discussions. In the short term I'll get an alpha version out for the next eLabFTW version so everyone can play with it on the demo and export/import `.eln` files and see if it plays well with other software. Medium term would be to ensure all members of the consortium have a working import/export implemented. Long term would be to promote that format everywhere so it becomes ubiquitous (for instance with data repositories).

**Comment by djeanner** [comment]
**- My background and expertise relevant to the goal of the ELNConsortium ...**

Expertise in NMR spectroscopy data (NMReDATA), former member of a IUPAC work group on spectroscopy. Collaborate with Mestrelab. Developer in C++ with basic knowledge on ontology, link data generation and reledata data modeling, _etc_.

**- My current activities and their relevance to the ELNConsortium ...**

Trying to push the CHEMeDATA initiative, aiming at annotating repository data to make them findable. Beyond data format and file content, focus on the need to provide essential information for OAI/PMH system or other search engine. 

**- My contribution to the ELNconsortium goal could be ...**

Work on a minimal ontology for chemistry (sample, analytical methods, etc.) as a kind of overhead of content descriptor with solid ontological basis.

**- My short-term activity within the ELNConsortium include ...**

I need to understand when the ELNConsortium will really do...
